src/components/EditListingWizard/EditListingWizard.css

```diff
@@ -85,14 +80,10 @@
   flex-grow: 1;
 
   @media (--viewportLarge) {
-    padding: 82px 15vw 82px 82px;
+    padding: 88px calc((100% - 1056px) / 2 + 36px) 82px 82px;
```

Comment:
Unfortunately, there are still problems with this.

`calc((100% - 1056px) / 2 + 36px)`
means that the right column will always be 36px narrower than Footer.
That's because `calc((100% - 1056px) / 2)` is already the filling the distance between browser window's right edge and 1056px wide content area.

However, we didn't think through what happens at the breakpoint. If the `window.innerWidth` is 1024px (aka `--viewportLarge`), then this calculation is actually `calc((1024px - 1056px) / 2)` which means `-16px` and negative padding is a faulty value for browsers and they (at least Chrome) discard it altogether.

This is rather annoying problem:
I couldn't think of any other solution than to add a new breakpoint for this component so that layout between **1024px** and **1128px** are covered differently than larger layouts. 
```css
@media (--viewportLarge) {
  padding: 88px 36px 82px 82px;
```
and 
```css
@media (min-width: 1128px) {
  padding: 88px calc((100% - 1056px) / 2) 82px 82px;
}
```

Similar breakpoint should be for the sidenav
`padding: 112px 0 82px 36px;` vs `padding: 112px 0 82px calc((100% - 1056px) / 2 + 74px);`

----

I made a quick search if we have used extra breakpoints anywhere and it seems that there are only 2 components that has wild breakpoints:
- **TopbarDesktop.css** uses `@media (min-width: 1024px) {` 
  - we could just change it to `@media (--viewportLarge) {`
- **ManageAvailabilityCalendar.css** uses `@media (min-width: 550px) {`
  - My recent addition since content in React-Dates calendar didn't work well with small table-cells.

Therefore, my suggestion is actually that we:
1) fix Topbar.css and 
2) add two new custom breakpoints:
  - `@custom-media --viewportSmall (min-width: 550px);`
  - `@custom-media --viewportLargeWithPaddings (min-width: 1128px);`